packages/@webex/plugin-cc/src/services/task/index.ts

```
@@ -141,15 +145,67 @@ export default class Task extends EventEmitter implements ITask {
     this.metricsManager = MetricsManager.getInstance();
     this.registerWebCallListeners();
+    this.setupAutoWrapUpTimer();
+  }
+
+  /**
+   * Sets up the automatic wrap-up timer if wrap-up is required
+   * @private
+   */
+  private setupAutoWrapUpTimer() {
+    if (this.data && this.data.wrapUpRequired) {
+      const wrapUpProps = this.wrapupProps.wrapUpProps;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)